### PR TITLE
DOC-5945for6.0: Added new ipv6 info

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Review by timofey-barmin — APPROVED** [review]

File: modules/install/pages/ipv6-setup.adoc [hunk 1]
```diff
@@ -22,14 +22,35 @@ See xref:install-intro.adoc[Installing the Couchbase Server Cluster] for more in
...
 
 . Stop the `couchbase-server` service.
 . In `/opt/couchbase/etc/couchbase/static_config`, set `ipv6` to `true`.
 . Delete `/opt/couchbase/var/lib/couchbase/config/config.dat`.
 . Restart the `couchbase-server` service (this starts Couchbase Server in IPv6 mode).
 To check, point a supported web browser to  `http://[::1]:8091`, which is the IPv6 address and port number on which Couchbase Server should be running.
 
+This concludes the procedure.
+
+=== Version 6.0.1
```
Comment (timofey-barmin) [inline modules/install/pages/ipv6-setup.adoc]: Perhaps we should and "and higher".
=== Version 6.0.1 and higher

File: modules/install/pages/ipv6-setup.adoc [hunk 2]
```diff
@@ -40,11 +61,32 @@ During installation of the Couchbase package, a check box is available to enable
...
-(Quit the Couchbase Server application if it is running.)
+(Quit the Couchbase Server application, if it is running.)
 . In `/Applications/Couchbase\ Server.app/Contents/Resources/couchbase-core/etc/couchbase/static_config.in`, set `ipv6` to `true`.
 . Delete `~/Library/Application\ Support/Couchbase/var/lib/couchbase/config/config.dat`.
 . Restart the `couchbase-server` service (this starts Couchbase Server in IPv6 mode).
 To check, point a supported web browser to  `http://[::1]:8091`, which is the IPv6 address and port number on which Couchbase Server should be running.
+
+This concludes the procedure.
+
+=== Version 6.0.1
```
Comment (timofey-barmin) [inline modules/install/pages/ipv6-setup.adoc]: Perhaps we should and "and higher".
=== Version 6.0.1 and higher

File: modules/install/pages/ipv6-setup.adoc (hunk 1 above)
Comment (timofey-barmin) [reply modules/install/pages/ipv6-setup.adoc]: On the other hand we don't want this to be used for 6.5.
We can use something like "and all next 6.0.* version"

Comment (tonyjhillman) [reply modules/install/pages/ipv6-setup.adoc]: ok

**Review by timofey-barmin — APPROVED** [review]

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**